Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: lint
+lint:
+	luacheck --config=.luacheckrc --no-unused-args --no-redefined \
+	                    ./icu-date.lua ./test
```

Comment:
Maybe add other lua files too?

Reply:
Done

The exception to this is the `spec` directory, because it seems to use luaunit that overrides global variables